jaeger_client/utils.py

```diff
@@ -22,25 +22,21 @@
 import struct
 import time
 
-from .metrics import Metrics
-
 
 class ErrorReporter(object):
     """
     Reports errors by emitting metrics, and if logger is provided,
     logging the error message once every log_interval_minutes
+
+    N.B. metrics will be deprecated in the future
     """
 
     def __init__(self, metrics, logger=None, log_interval_minutes=15):
```

Comment:
I suggest `metrics=None` - keep it for backwards compatibility, but don't require it.